lib/cocoapods-core/specification/root_attribute_accessors.rb

```diff
@@ -40,6 +40,15 @@ def version
           end
         end
 
+        # @return [Array<Requirement>] The swift_versions required to use the
+        #                              specification.
+        #
+        def swift_versions
+          @swift_versions ||= if versions = attributes_hash['swift_versions']
```

Comment:
do we need the `if versions = attributes_hash['swift_versions']`

Does it also need to be set in the `cocoapods_versions` method or does that have a default value always set?

Reply:
I think `Requirement` initializer covers the case of `nil` or empty string.

Reply:
but i guess we would prefer a `nil` value than a `Requirement` value with `<Pod::Version version=0>`.

Reply:
I prefer a nil here for the objective-c pod case

Reply:
nil here is the right call